### ISSUE TITLE
Error Running simple example

### ISSUE THREAD

**Issue by padmarc** [fkooman/php-oauth-client#47]
Hello,
I'm trying to start the simple example server and gdrive example. I configured the endpoints and other parameters but I get the same type of error in both cases. 

Argument 1 passed to Guzzle\\Common\\AbstractHasDispatcher::addSubscriber() must implement interface Symfony\\Component\\EventDispatcher\\EventSubscriberInterface, instance of fkooman\\Guzzle\\Plugin\\BearerAuth\\BearerAuth given, called in /var/www/oauth-client2/php-oauth-client/example/simple/index.php on line 35

**Comment by fkooman** [comment]
Yeah it broken when the Bearer guzzle plugin got updated for Guzzle 5.x unfortunately. I'll have to look into that soon!

**Comment by padmarc** [comment]
OK, Thanks for your answer!
